sdk/lib/ucrt/startup/exit.cpp

```diff
@@ -285,21 +285,25 @@ extern "C" int __cdecl _is_c_termination_complete()
 extern "C" void __cdecl exit(int const return_code)
 {
     common_exit(return_code, _crt_exit_full_cleanup, _crt_exit_terminate_process);
+    UNREACHABLE;
```

Comment:
This UNREACHABLE cannot be added in the `common_exit` instead? (unless `common_exit` is expected to return...)

Reply:
No, because `common_exit` does return in some cases:
```
extern "C" void __cdecl _cexit()
{
    common_exit(0, _crt_exit_full_cleanup, _crt_exit_return_to_caller);
}

```